### PR TITLE
added full documentation and .gitignore updates

### PR DESCRIPTION
**Pull request by KarimovMurodilla**
Writed documentation using mkdocs. Changed .gitignore. Updated setup version to 3.1 (and also in PyPi)

### CODE REVIEW

File: docs/generate_url.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Payme ning to'lov qismiga foydalanuvchini yuborish uchun, URL ni yaratib olishimiz kerak bo'ladi
+
+> Namuna
+![alt text](https://imgur.com/IE0yV4Z.jpg)
+
+```python
+from clickuz import ClickUz
+
+url = ClickUz.generate_url(order_id='123456789',amount='100000')
```
Comment (begyy) [inline docs/generate_url.md]: bu clickniku bizaga paymeniki kere :D

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+![alt text](https://img.shields.io/badge/code%20style-black-000000.svg)
+[![Downloads](https://img.shields.io/pypi/v/clickuz)](https://pypi.org/project/ClickUz)
+[![Downloads](https://black.readthedocs.io/en/stable/_static/license.svg)](https://github.com/begyy/ClickUz/blob/master/LICENSE)
+[![Downloads](https://img.shields.io/badge/docs-passing-green)](https://begyy.github.io/ClickUz/)
+[![Downloads](https://img.shields.io/badge/telegram-djangouzb-green)](https://t.me/djangouzb)
+[![Downloads](https://img.shields.io/badge/author-begyy-green)](https://t.me/begyy)
+<div class="termy">
+
+```console
+pip install clickuz
```
Comment (begyy) [inline docs/index.md]: pip install paycomuz

File: docs/generate_url.md (hunk 1 above)
Comment (begyy) [reply docs/generate_url.md]: image

Comment (KarimovMurodilla) [reply docs/generate_url.md]: > bu clickniku bizaga paymeniki kere :D

almashtirish esdan chiqibti xD

**Comment by KarimovMurodilla** [comment]
fixed

**Comment by begyy** [comment]
@KarimovMurodilla merge qildm endi mcdocs bilan deploy qivoring sizniyam proektga qoshib qoydm access bor
Salom hayot la blan)

**Comment by KarimovMurodilla** [comment]
> @KarimovMurodilla merge qildm endi mcdocs bilan deploy qivoring sizniyam proektga qoshib qoydm access bor Salom hayot la blan)

Assalomu aleykum. Raxmat sizniyam o'tgan hayitiz bilan. Aha bo'ldi, qilib qo'yaman

**Comment by KarimovMurodilla** [comment]
https://begyy.github.io/PaycomUz/
deploy qildim, lekin negadir haliyam 404 turibti

**Pull request merged**